includes/Generator/Coupon.php

```diff
@@ -25,12 +25,15 @@ public static function generate( $save = true, $assoc_args = array() ) {
 		parent::maybe_initialize_generators();
 
 		$defaults = array(
-			'min' => 5,
-			'max' => 100,
+			'min'           => 5,
+			'max'           => 100,
+			'discount_type' => '',
```

Comment:
I was initially confused by this. Would it be helpful to have a comment explaining the WooCommerce defaults to `fixed_cart` if no type is provided?